File: Laba_1_java/src/BreakerElementInformation.java
public class BreakerElementInformation implements Comparable<BreakerElementInformation> {
    private Integer _calculationTime_ms;
    private Integer _id;
    private Boolean _isStop;

    public BreakerElementInformation(Integer calculationTime_ms, Integer id){
        _calculationTime_ms = calculationTime_ms;
        _id = id;
        _isStop = false;
    }

    public void Stop(){
        _isStop = true;
    }

    public Integer GetCalculationTime_ms(){
        return _calculationTime_ms;
    }

    public Integer GetId(){
        return _id;
    }

    public Boolean GetIsStop(){
        return _isStop;
    }

    @Override
    public int compareTo(BreakerElementInformation el) {
        return _calculationTime_ms.compareTo(el.GetCalculationTime_ms());
    }
}

File: Laba_1_sharp/Laba_1/BreakerElementInformation.cs
using System;

namespace Laba_1
{
    public class BreakerElementInformation : IComparable<BreakerElementInformation>
    {
        public int CalculationTime { get; private set; }
        public int Id { get; private set; }

        public bool IsStop { get; private set; }

        public BreakerElementInformation(int calculationTime, int id, bool isStop = false)
        {
            CalculationTime = calculationTime;
            Id = id;
            IsStop = isStop;
        }

        public void Stop()
        {
            IsStop = true;
        }

        public int CompareTo(BreakerElementInformation other)
        {
            return CalculationTime.CompareTo(other.CalculationTime);
        }
    }
}